docs/self-managed/operational-guides/production-guide.md

```diff
@@ -0,0 +1,530 @@
+
+## Security Guidelines
+
+Here are some points to keep in mind when considering security:
+
+- To limit unnecessary permissions, and reduce risk of token exploitations, it is recommended to disable auto-mounting of the default Service Account. It is possible to do so by adding the following to your `production-values.yaml`:
+
+```yaml
+identity:
+  serviceAccount:
+    enabled: false
+console:
+  serviceAccount:
+    enabled: false
+webModeler:
+  serviceAccount:
+    enabled: false
+connectors:
+  serviceAccount:
+    enabled: false
+core:
+  serviceAccount:
+    enabled: false
+optimize:
+  serviceAccount:
+    enabled: false
+```
+
+You should only enable the auto-mounting of a service account token when the application explicitly needs access to the Kubernetes API server or you have created a service account with the exact permissions required for the application and bound it to the pod.
+
+- If you have a use case for enabling [Network Policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/) then it is recommended to do so.
+<!--Maybe link this to customer: https://github.com/ahmetb/kubernetes-network-policy-recipes-->
+- It is possible to have a pod security standard that is suitable to the security constraints you might have. This is possible through modifying the Pod Security Admission. Please refer to the [Kubernetes documentation](https://kubernetes.io/docs/concepts/security/pod-security-admission/) in order to do so.
+- By default, the Camunda Helm Chart, uses a read-only root file-system configuration for the pod. It is recommended to keep this default. No addition needs to be made in your `production-values.yaml`.
+- Disable privileged containers
+- It is possible to modify either the `containerSecurityContext` or the `podSecurityContext`. For example, here is a configuration for the core component that can be added to your `production-values.yaml`:
+
+```yaml
+podSecurityContext:
+  runAsNonRoot: true
+  fsGroup: 1001
+  seccompProfile:
+    type: RuntimeDefault
+
+containerSecurityContext:
+  allowPrivilegeEscalation: false
+  privileged: false
+  readOnlyRootFilesystem: true
+  runAsNonRoot: true
+  runAsUser: 1001
+  seccompProfile:
+    type: RuntimeDefault
+```
+
+- It is best to only pull from a private registry such as [Amazon ECR](https://aws.amazon.com/ecr/) and not pull directly from Docker Hub. This is because it will increase the control you have over the images, will save you from rate limits, and increase performance and reliability. Furthermore, it is possible to restrict your cluster to only pull from known registries, this can be done using [Open Policy Agent](https://blog.openpolicyagent.org/securing-the-kubernetes-api-with-open-policy-agent-ce93af0552c3#3c6e), for example.
```

Comment:
It is recommended to pull images exclusively from a private registry, such as [Amazon ECR](https://aws.amazon.com/ecr/), rather than directly from Docker Hub. Doing so enhances control over the images, avoids rate limits, and improves performance and reliability. Additionally, you can configure your cluster to pull images only from trusted registries. Tools like [Open Policy Agent](https://blog.openpolicyagent.org/securing-the-kubernetes-api-with-open-policy-agent-ce93af0552c3#3c6e) can be used to enforce this restriction.